src/shipit_frontend/.neutrinorc.js

```diff
@@ -65,6 +77,10 @@ module.exports = {
       }
     ],
     ['@neutrinojs/env', Object.keys(envs)],
+    (neutrino) => {
+        neutrino.config.when(process.env.NODE_ENV === 'production', config => {
+            config.devtool('source-map');
+        });
+    }
   ]
-  // TODO: add source-map, see https://github.com/mozilla/firefox-code-coverage-frontend/commit/36f362f72667e2f309b43b23a84e6db14266b21a
```

Comment:
Thank you!

Reply:
I added this but I'm still not seeing source-map kick in. I guess we need to come back to this.